decidim-elections/app/models/decidim/elections/election.rb

```diff
@@ -78,6 +78,20 @@ def minimum_answers?
         questions.all? { |question| question.answers.size > 1 }
       end
 
+      # # Public: Checks if the election results are published and election finished
+      # #
+      # # Returns a boolean.
+      def results_published?
+        bb_status == "results_published"
```

Comment:
I like to keep these magic strings as class constants. It's even better if you need the value in different places but if it's just in here maybe it's not worth it. What do you think?

Reply:
I'm not using the strings anymore. 😄